### ISSUE TITLE
Top of tree does not work

### ISSUE THREAD

**Issue by jacobweinstock** [equinix-labs/terraform-equinix-metal-eks-anywhere#73]
The current top of tree(553ed65327cb3b2bdb9d2adee914e04d1ffef861) does not work. It fails while waiting for cloud-init to complete on the admin node.

```bash
module.eksa.null_resource.wait_for_cloud_init (remote-exec): status: error
╷
│ Error: remote-exec provisioner error
│ 
│   with module.eksa.null_resource.wait_for_cloud_init,
│   on ../../main.tf line 184, in resource "null_resource" "wait_for_cloud_init":
│  184:   provisioner "remote-exec" {
│ 
│ error executing "/tmp/terraform_1247039134.sh": Process exited with status 1
```

**Comment by jacobweinstock** [comment]
CC @displague , @cprivitere

**Comment by cprivitere** [comment]
So this to me appears to be random depending on whether the google deb repos for kubernetes are responding on their ipv6 addresses.

**Comment by displague** [comment]
This has been corrected by #72 and #74.
